src/main/java/uk/gov/hmcts/reform/cwrdapi/domain/CaseWorkerAudit.java

```diff
@@ -14,17 +14,21 @@
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.EntityListeners;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.NamedAttributeNode;
 import javax.persistence.NamedEntityGraph;
 import javax.persistence.OneToMany;
+import javax.persistence.SequenceGenerator;
 import javax.validation.constraints.Size;
 
 @Entity(name = "case_worker_audit")
 @Getter
 @Setter
 @NoArgsConstructor
 @EntityListeners(AuditingEntityListener.class)
+@SequenceGenerator(name = "job_id_seq", sequenceName = "job_id_seq", allocationSize = 1)
 @NamedEntityGraph(
         name = "CaseWorkerAudit.alljoins",
```

Comment:
remove for now. I think not required now @NamedEntityGraph(
        name = "CaseWorkerAudit.alljoins",	        name = "CaseWorkerAudit.alljoins",